src/app/styles/charts.scss

```diff
@@ -0,0 +1,9 @@
+.chart {
```

Comment:
file name should match class name -- thus `chart.scss` not `charts.scss`

Reply:
fixed